base/applications/mspaint/toolsmodel.h

```diff
@@ -41,13 +70,18 @@ class ToolsModel
     int m_rubberRadius;
     BOOL m_transpBg;
     int m_zoom;
+    ToolBase *m_pTools[TOOL_MAX + 1];
+    ToolBase *m_pToolObject;
+
+    ToolBase *fetchToolObject(TOOLTYPE nTool);
```

Comment:
`GetOrCreate` might convey the implementation better?

Reply:
OK.